### PR TITLE
Drawing Classifier User Guide Polish

### LINKED ISSUE #1529: Drawing classifier - Userguide need editing
- Tildes should be removed as they don't work from the two scripts given `quickdraw_dir = '~/Downloads/quickdraw'` and `quickdraw_dir = '~/Downloads/quickdraw'`

- The script given to build an SFrame doesn't return anything

- sf = build_stroke_sframe() should be sf = build_strokes_sframe()

### LINKED ISSUE #1558: Drawing Classification README doesn't specify expected SFrame format
The data-preparation file can discuss how to convert data into the expected format, but the expected format itself should be specified on the main page

### PR DESCRIPTION
**Pull request by shantanuchhabra**
Fixes #1529 
Fixes #1558

### CODE REVIEW

File: userguide/drawing_classifier/data-preparation.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ both as bitmaps and as stroke-based drawings.
 Open a new terminal window and run the following commands:
 
 ```
-$ mkdir -p ~/Downloads/quickdraw
-$ cd ~/Downloads/quickdraw
+$ mkdir -p /Users/$(whoami)/Downloads/quickdraw
```
Comment (nickjong) [inline userguide/drawing_classifier/data-preparation.md]: I don't think this generalizes to other platforms

Comment (shantanuchhabra) [reply userguide/drawing_classifier/data-preparation.md]: Doesn't `whoami` work on both macOS and Linux?

File: userguide/drawing_classifier/README.md [hunk 2]
```diff
@@ -16,6 +16,40 @@ In this example, we use data for two of the 345 classes from "Quick,Draw!" --
 square and triangle. Go to [Data Preparation](data-preparation.md) to create the 
 `square_triangle.sframe` that we will use in the introductory example.
 
+The feature in the input SFrame to the Drawing Classifier can have the following
+two formats:
+
+1. Bitmap-based drawings (`dtype=turicreate.Image`): Each bitmap-based drawing
+must be represented as a grayscale image of any size. The network takes in
```
[the remark this answers is not in the document]
Comment (shantanuchhabra) [reply userguide/drawing_classifier/README.md]: We don't at the moment -- we require the input to be grayscale. Tracking this issue [here](#1594).

File: userguide/drawing_classifier/data-preparation.md (hunk 1 above)
Comment (nickjong) [reply userguide/drawing_classifier/data-preparation.md]: Is it safe to assume that we should write to /Users?

Could we just use the current working directory here, for maximum generality?

File: userguide/drawing_classifier/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nickjong) [reply userguide/drawing_classifier/README.md]: The link you posted doesn't seem to go anywhere.

I believe this line resamples the image to one channel (grayscale): https://github.com/apple/turicreate/blob/23e1e8be55f18580939df768c6561c2e22e48ad8/src/unity/python/turicreate/toolkits/drawing_classifier/_sframe_loader.py#L65

[the remark this answers is not in the document]
Comment (shantanuchhabra) [reply userguide/drawing_classifier/README.md]: You're right, that does resample the image to one channel.

File: userguide/drawing_classifier/data-preparation.md (hunk 1 above)
Comment (shantanuchhabra) [reply userguide/drawing_classifier/data-preparation.md]: Done.

File: userguide/drawing_classifier/data-preparation.md [hunk 3]
```diff
@@ -15,11 +15,11 @@ First, we will download all the data points in the "Quick, Draw!" dataset for
 the "square" and "triangle" classes -- around 120,000 examples each, 
 both as bitmaps and as stroke-based drawings.
 
-Open a new terminal window and run the following commands:
+Open a new terminal window, go to your working directory and run the following commands:
 
 ```
-$ mkdir -p ~/Downloads/quickdraw
-$ cd ~/Downloads/quickdraw
+$ mkdir -p quickdraw
```
Comment (syoutsey) [inline userguide/drawing_classifier/data-preparation.md]: There's no need for the `-p` flag anymore.

Comment (shantanuchhabra) [reply userguide/drawing_classifier/data-preparation.md]: Fixed.

**Pull request merged**